tests/integration_tests/build/test_build.py

```diff
@@ -12,42 +12,29 @@
 
 MACHINE = platform.machine()
 FEATURES = [""]
-BUILD_TYPES = ["debug", "release"]
-
 TARGETS = ["{}-unknown-linux-gnu".format(MACHINE),
            "{}-unknown-linux-musl".format(MACHINE)]
 
 
 @pytest.mark.parametrize(
-    "features, build_type, target",
-    itertools.product(FEATURES, BUILD_TYPES, TARGETS)
+    "features, target",
```

Comment:
Not related to this commit, but we don't have features any more. Even the array is empty. The parametrize should only receive the targets.

Reply:
I thought a bit on that, but hearing @alexandruag in my head:
> Whether you like'em or not, `#feature`s are coming boys!

I decided to keep the capability 😄